creator-node/src/index.ts

```diff
@@ -183,6 +201,17 @@ const startAppForWorker = async () => {
   logger.info('Initialized app and server')
   await serviceRegistry.initServicesThatRequireServer(appInfo.app)
 
+  cluster.worker!.on('message', (msg) => {
+    if (msg?.cmd === 'setSpecialWorkerId') {
```

Comment:
yep! it makes more sense for the worker to register its own listener instead of the primary registering a listener for all workers. this way we don't need to worry about re-registering events when workers respawn